app/index.html

```diff
@@ -10,9 +10,18 @@
     <link rel="stylesheet" href="styles/compiled.css">
     <!-- endbuild -->
   </head>
-  <body>
-    <h1>Chronicle</h1>
-    <h2>Welcome home</h2>
-    <a href="/auth/login">Sign in</a>
+  <body class="l-flex-body">
+    <div id="container" class="row">
+      <div class="row-inner">
+        <h1>Chronicle</h1>
+        <h2>Welcome home</h2>
+        <a href="/auth/login">Sign in</a>
+      </div>
+    </div>
+    <footer id="global-footer" class="row">
+      <div class="row-inner">
+        <p>&copy; 2015 &bull; <a href="http://mozilla.org/" target="_blank">Mozilla</a></p>
+      </div>
+    </footer>
```

Comment:
Since this footer seems to be shared between the app/index.html and app/app.html, not sure if it can be recycled similar to the `<!-- view: global_header/global_header -->` for the app.html global header.